srcpkgs/darktable/template

```diff
@@ -1,34 +1,36 @@
 # Template file for 'darktable'
 pkgname=darktable
-version=3.0.2
+version=3.2.1
 revision=1
 # upstream only supports these archs:
 archs="x86_64* aarch64* ppc64le*"
 build_style=cmake
 # this makes sure to use -march=generic and -msse3
 configure_args="-DBINARY_PACKAGE_BUILD=ON -DBUILD_NOISE_TOOLS=ON
  -DRAWSPEED_ENABLE_LTO=ON"
-hostmakedepends="pkg-config intltool libxslt-devel"
+hostmakedepends="pkg-config intltool libxslt-devel desktop-file-utils"
 makedepends="gtk+3-devel glib-devel exiv2-devel libglade-devel libxslt-devel
  dbus-glib-devel libcurl-devel libgphoto2-devel libwebp-devel libsoup-devel
  lensfun-devel sqlite-devel librsvg-devel lua53-devel json-glib-devel
  libgomp-devel libopenjpeg2-devel libopenexr-devel libgraphicsmagick-devel
- libsecret-devel pugixml-devel libosmgpsmap-devel colord-gtk-devel
- desktop-file-utils"
+ libsecret-devel pugixml-devel libosmgpsmap-devel colord-gtk-devel"
 depends="adwaita-icon-theme"
 short_desc="Virtual lighttable and darkroom for photographers"
 maintainer="lemmi <lemmi@nerd2nerd.org>"
 license="GPL-3.0-or-later"
 homepage="http://www.darktable.org/"
 changelog="https://github.com/darktable-org/darktable/releases"
 distfiles="https://github.com/darktable-org/darktable/releases/download/release-${version}/darktable-${version}.tar.xz"
-checksum=6abaf661fe9414e92bdb33b58b98ef024ccf6132b7876abaf0751ec2109f36fb
+checksum=6e3683ea88dc0a0271be7eca4fd594b9e46b1b7194847825a8d0a0c12bdeb90c
 
 # Temporarily disable OpenMP for aarch64:
 #
 # see https://github.com/void-linux/void-packages/issues/21105
 
 case $XBPS_TARGET_MACHINE in
+	x86_64*)
+		makedepends+=" gmic-devel"
+		;;
```

Comment:
Wouldn't this be for `if [ "$CROSS_BUILD" ]`, instead? Also potentially a build option. and you select the default ones based on it being a cross build or not, since `gmic` adds quite a few features?

Reply:
Yep, checking for cross is probably the correct solution. Thanks